### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Akianonymus**

### CODE REVIEW

**Comment by labbots** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6001654&bid=19676524)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -148,6 +148,10 @@ _check_connection() {
 -   Add a functions only if you are going to use it multiple times, otherwise use the code directly, exceptions can be made where it can make the script messier.
 -   For more info, start from [tldp guide](https://www.tldp.org/LDP/Bash-Beginners-Guide/html/chap_01.html).
 
+Additional rules to follow:
+
+-  Always use force write ( >| ), even in case of writing to /dev/null
```
Comment (labbots) [inline CONTRIBUTING.md:153]: Codacy found an issue: [[list-item-indent] Incorrect list-item indent: remove 1 space](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=6001654)

**Comment by labbots** [comment]
@Akianonymus please raise a PR before merging any code to master. Please do not make changes directly to master without PR and kindly wait for approval before merging.

**Comment by Akianonymus** [comment]
> @Akianonymus please raise a PR before merging any code to master. Please do not make changes directly to master without PR and kindly wait for approval before merging.

Well, this is a PR to my extent of my knowledge. I did not push changes directly to master. The checks passed too, along with my personal tests.

What i did not do was wait for you to approve, that's my mistake. I will keep that in mind for future PRs.

**Comment by labbots** [comment]
@Akianonymus I was not talking about this PR. There were some commits in master in the past. So I was just asking you to create PR for any changes in the future so that I could review the changes.

**Comment by Akianonymus** [comment]
@labbots

I saw, https://github.com/labbots/google-drive-upload/commit/f73c539410946d16b6cd8645aded5c69cd9ea726

Just that "commit".

I thought it was not significant to create a PR for such a small change which definitely won't do any harm and is 100% test.

Anyway, will do from next time.

**Pull request merged**